src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -66,16 +70,50 @@ class FrontendGenImpl {
       const onnx::NodeProto &);
 
   std::map<std::string, ImportHandlerType> import_handler_map_;
+  std::map<std::string, int64_t> opset_map_;
+  std::map<std::string, onnx::ValueInfoProto> value_info_map;
```

Comment:
Could you explains what these data structures do?  Would be nice to have a comment in the code about it.

Reply:
Added comment to the source file.